aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -1542,18 +1542,25 @@ DropoutState& get_dropout_state(double dropout_p, bool train, TensorOptions opti
 
 Tensor try_get_weight_buf(
       const Tensor& input, TensorList parameters, bool has_biases,
-      cudnnRNNMode_t mode, int64_t hidden_size, int64_t proj_size, int64_t num_layers, bool bidirectional) {
+      cudnnRNNMode_t mode, c10::SymInt hidden_size, c10::SymInt proj_size, int64_t num_layers, bool bidirectional) {
```

Comment:
LSTM-related changes here along with the meta registrations are being merged to master in #90701.